lottie-swift/src/Public/Animation/AnimationView.swift

```diff
@@ -402,6 +402,7 @@ final public class AnimationView: LottieView {
   
   /// Reloads the images supplied to the animation from the `imageProvider`
   public func reloadImages() {
+    animationLayer?.imageProvider = imageProvider
```

Comment:
This should go in the ImageProvider setter method. Up around line 94.

```
 public var imageProvider: AnimationImageProvider {
    didSet {
    animationLayer?.imageProvider = imageProvider
      reloadImages()
    }
  }
```

Reply:
Done :)